<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>qtMapCoordinatesWidget</class>
 <widget class="QWidget" name="qtMapCoordinatesWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>441</width>
    <height>46</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="0" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="latLabel">
       <property name="text">
        <string>Latitude</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="LatitudeEdit">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>1</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lonLabel">
       <property name="text">
        <string>Longitude</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="LongitudeEdit">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>1</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Zoom:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="ZoomEdit">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>12</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
